### ISSUE TITLE
Storage Scanner Missing Items highlight

### ISSUE THREAD

**Issue by Thewms** [McJtyMods/RFToolsStorage#32]
Currently playing in 1.16.4
Request: With older versions of the Storage Scanner when you would try to craft something from the terminal and were missing an ingredient, it would highlight that item with a red box. 
Example: using the 3x3 grid in the terminal to craft a stone pick axe. the 2 sticks in the recipe turn red because I have no sticks in the system.
This was incredibly helpful to be able to quickly see what was still needed in a recipe or to realize when you've run out of something instead of having to search each piece of a recipe to figure out what you are missing. Any chance this feature will be added back in?

**Comment by McJty** [comment]
yes I'm planning to fix that. I have to check out why it got broken but it certainly should be there
